docs/modules/ROOT/nav.adoc

```diff
@@ -157,7 +157,11 @@ include::cp-subsystem:partial$nav.adoc[]
 * xref:storage:high-density-memory.adoc[]
 include::tiered-storage:partial$nav.adoc[]
 * xref:cluster-performance:thread-per-core-tpc.adoc[]
-include::data-connections:partial$nav.adoc[]
+* Data Connections
```

Comment:
As discussed on Slack, would be best to put in `Develop and Build`

Reply:
What do you think of it under the pipelines section as per the latest commit?

Reply:
Perfect 👌